### DISCUSSION TITLE
Docs website is throwing 502 errors

### DISCUSSION THREAD

**Discussion by slipperystairs** [apexcharts/apexcharts.js#4611] [General]
I can't seem to navigate to a majority of the pages for the docs without getting a 502, other websites are working just fine for me. Is anyone else experiencing this?

![image](https://github.com/user-attachments/assets/4d32f082-11c3-42ad-b2f4-332eb5c25bca)

**Comment by brianlagunas** [comment]
The website is down. We're looking into it.

**Reply by slipperystairs** [reply]
That is what I figured, but I wanted to make sure you guys were aware. Thanks for looking into it!

**Comment by brianlagunas** [comment]
Website is back up. It may take a while to reach the rest of the world.
